src/modules/core/components/image-selector/image-selector.module.scss

```diff
@@ -0,0 +1,114 @@
+@use '@/styles/utils';
+@use '@/styles/variables.module';
+
+.container {
+  position: relative;
+  width: 150px;
+  height: 150px;
+  border-radius: 21px;
+  border: 1px dashed variables.$primary-light-color;
+  background: variables.$grey-light-color;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition: border variables.$transition-ultrafast ease-in-out;
```

Comment:
we probably need to add `cursor: pointer` when image isn't selected since all container is clickable